<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_Texture</class>
 <widget class="QMainWindow" name="MainWindow_Texture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow_Texture</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1109</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionSet_Texture_Dir"/>
    <addaction name="actionSet_obj_dir"/>
    <addaction name="actionLoad_Obj"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>view</string>
    </property>
    <addaction name="actionShow_data_base"/>
    <addaction name="actionData_dock"/>
    <addaction name="actionShow_line"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>edit</string>
    </property>
    <addaction name="actionClear_Select"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_Texture_Brow">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>261</width>
       <height>791</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_Texture_Brow"/>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar_Texture">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>16</width>
       <height>791</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>321</width>
       <height>771</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_Obj_Brow"/>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar_Objs">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>0</y>
       <width>16</width>
       <height>771</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Shape_Texture">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun_d0_synthesis"/>
   <addaction name="actionRun_d0_all"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionRun_d1_synthesis"/>
   <addaction name="actionRun_d1_all"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionSceneManipulation"/>
   <addaction name="separator"/>
   <addaction name="actionArea_Select"/>
   <addaction name="separator"/>
   <addaction name="actionArea_Select_Only_Visible"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_all_unselected"/>
  </widget>
  <action name="actionSet_Texture_Dir">
   <property name="text">
    <string>set_texture_dir</string>
   </property>
  </action>
  <action name="actionShow_data_base">
   <property name="text">
    <string>show_data_base</string>
   </property>
  </action>
  <action name="actionData_dock">
   <property name="text">
    <string>data_dock</string>
   </property>
  </action>
  <action name="actionLoad_Obj">
   <property name="text">
    <string>load_obj</string>
   </property>
  </action>
  <action name="actionRun_d1_synthesis">
   <property name="text">
    <string>run_d1_synthesis</string>
   </property>
  </action>
  <action name="actionRun_d0_synthesis">
   <property name="text">
    <string>run_d0_synthesis</string>
   </property>
  </action>
  <action name="actionArea_Select_Only_Visible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Area_Select_Only_Visible</string>
   </property>
  </action>
  <action name="actionClear_Select">
   <property name="text">
    <string>clear_select</string>
   </property>
  </action>
  <action name="actionSceneManipulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SceneManipulation</string>
   </property>
  </action>
  <action name="actionArea_Select">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Area_Select</string>
   </property>
  </action>
  <action name="actionRun_d0_all">
   <property name="text">
    <string>run_d0_all</string>
   </property>
  </action>
  <action name="actionRun_d1_all">
   <property name="text">
    <string>run_d1_all</string>
   </property>
  </action>
  <action name="actionSet_obj_dir">
   <property name="text">
    <string>set_obj_dir</string>
   </property>
  </action>
  <action name="actionSelect_all_unselected">
   <property name="text">
    <string>select_all_unselected</string>
   </property>
  </action>
  <action name="actionShow_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>show_line</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
